_posts/2018-09-29-cfp-in-progress.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Call for Proposals in progress, keynotes announcement"
+date: 2018-10-06
+authors:
+  - apiraino
+---
+
+# Call for Proposals
+
+Last week we closed the CfP (Call for Proposals) submission window, we allowed a couple more days to really give everyone a chance to submit.
+
+We received 50 interesting proposals to fill up the talks slots for the upcoming Rustfest in Rome, thank you so much! Now it's our turn to open the envelopes and choose those that will make to the conference. As usual, the choice will be tough, all the topics submitted looks absolutely .
+
+Like every year, there's a preliminary selection; here is a draft of the schedule, to be refined and finalized:
+
+| Talk type     | amount planned |
+| ------------- | --------------:|
+| Keynotes      | 2              |
+| 15min format  | 2              |
+| 30min format  | 7              |
+| 45min format  | 2              |
```

Comment:
The keynotes are planned as 40min, but there are no 45min slots.

Reply:
ok!